### PR TITLE
[stable/external-dns] fix for issue 20217, added aws-evaluate-target-health flag

### PR DESCRIPTION
**Pull request by javamonkey79**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/external-dns/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.14.3
+version: 2.14.4
```
Comment (carrodher) [inline stable/external-dns/Chart.yaml]: As it is a new feature and not bug fixing I would increase this in a minor
```suggestion
version: 2.15.0
```

**Review by carrodher — CHANGES_REQUESTED** [review]
Thanks for the contribution. You need to add the parameter in the proper section in the _values.yaml_ and _values-production.yaml_.

Context: https://github.com/helm/charts/blob/master/stable/external-dns/values.yaml#L74

**Comment by juan131** [comment]
- fixes https://github.com/helm/charts/issues/20217

File: stable/external-dns/Chart.yaml (hunk 1 above)
Comment (javamonkey79) [reply stable/external-dns/Chart.yaml]: Done.

**Review by javamonkey79 — COMMENTED** [review]
Done

**Review by sameersbn — APPROVED** [review]
/lgtm

**Pull request merged**